unstructured/partition/pdf.py

```diff
@@ -363,6 +364,38 @@ def _process_pdfminer_pages(
     return elements
 
 
+def convert_pdf_to_images(
+    filename: str = "",
+    file: Optional[Union[bytes, BinaryIO, SpooledTemporaryFile]] = None,
+    chunk_size: int = 10,
+) -> Iterator[List[object]]:
+    # Convert a PDF in small chunks of pages at a time (e.g. 1-10, 11-20... and so on)
```

Comment:
```suggestion
    ```Convert a PDF in small chunks of pages at a time (e.g. 1-10, 11-20... and so on)```
```

Reply:
LGTM!